src/main/flight/imu.c

```diff
@@ -634,3 +634,12 @@ bool isUpright(void)
     return true;
 #endif
 }
+
+bool isUpsidedown(void)
+{
+#ifdef USE_ACC
+    return !sensors(SENSOR_ACC) || (attitudeIsEstablished && getCosTiltAngle() < 0);
```

Comment:
This says the heli is upside-down if ACC is not enabled. This can't be right?
Please fix.

Reply:
Should be:
```
return sensors(SENSOR_ACC) && attitudeIsEstablished && getCosTiltAngle() < 0;
```